openstf/metrics/metrics.py

```diff
@@ -48,7 +48,11 @@ def r_mae(realised, forecast):
     The range is based on the load range of the previous two weeks"""
 
     # Determine load range on entire dataset
-    range_ = realised.max() - realised.min()
+    range_ = (
```

Comment:
SonarCloud complains that we have some duplicated code here. It will probably also improve readability to  move the determination of the range and setting it to np.nan under some conditions to a separate function.